index.bs

```diff
@@ -6571,7 +6571,9 @@ and the corresponding value is the <dfn>authenticator extension output</dfn> for
 
 <dfn>Unsigned extension outputs</dfn> are represented independently from [=authenticator data=] and returned by authenticators
 as a separate map, keyed with the same [=extension identifier=]. This map only contains entries for authenticator
-extensions that make use of unsigned outputs.
+extensions that make use of unsigned outputs. Unsigned outputs are useful when extensions output a signature over
+the [=authenticator data=] (because otherwise a signature would have to sign over itself, which isn't possible) or when
+the extension outputs should not be sent to the RP.
```

Comment:
Are there any examples currently in the spec of extension output that **should not** be sent to the RP? I've never really considered this as a possibility before.

Reply:
The PRF extension.

Reply:
I didn't appreciate that this is was about an interaction between Client and Authenticator and isn't quite related to e.g. client extension outputs that RPs ultimately see.